cfme/tests/test_rest_manual.py

```diff
@@ -94,3 +66,148 @@ def test_notification_url_parallel_requests():
         1700378
     """
     pass
+
+
+@pytest.mark.manual
+@pytest.mark.meta(coverage=[1761836])
+@pytest.mark.tier(3)
+@pytest.mark.parametrize("implementation", ["UI", "Rest"])
```

Comment:
There are already defined appropriate classes. and please use `context` instead for better readability
`@pytest.mark.parametrize('context', [ViaSSUI, ViaUI])`

Reply:
@valaparthvi I think there was a misunderstanding here - there are actual ViaUI / ViaRest classes that should be parametrized against here, and those context classes can be used directly.  Your update still just parametrizes the strings.

Example:
https://github.com/ManageIQ/integration_tests/blob/d752d1822102c5a8c246babf1b07b95e5d0d1ec5/cfme/tests/cloud/test_instance_reconfigure.py#L18

Reply:
I understood that, but um okay here's my question: If I use the classes instead of strings, will I also need to use those contexts? Because I am only using this parametrization so that I know which way to verify my response. Should I stil use the classes?

Reply:
@valaparthvi, you aren't required to use these classes in test if it isn't necessary. It will only be used for generating tests.